Resources/Public/JavaScript/Src/bootstrap.stickyheader.js

```diff
@@ -1,12 +1,7 @@
-$(function () {
-
-    $(window).on('resize, scroll', function () {
-        var stickyheaderScrolled = $(window).scrollTop();
-        if (stickyheaderScrolled > 150) {
-            $('.navbar-fixed-top').addClass('navbar-transition');
-        } else {
-            $('.navbar-fixed-top').removeClass('navbar-transition');
-        }
-    });
-
+var navelem = document.getElementsByClassName("navbar-fixed-top");
```

Comment:
Do we need such a shortened variable name here, what about something like `stickyheader` (like before) or `navigation` instead?